src/components/InputGroup/index.tsx

```diff
@@ -31,7 +32,7 @@ export const getFlexSettings = (childCount, userSettings) => {
  * @param {Object} props react props
  * @returns {ReactElement}
  */
-const InputGroup = ({ children, flexSettings }) => {
+const InputGroup = ({ children, flexSettings }: InputGroupProps): React.ReactNode => {
```

Comment:
This can probably just be `JSX.Element` as you know it always will return a single one of them?